charts/cluster-scanner/README.md

```diff
@@ -0,0 +1,5 @@
+# Cluster Scanner
+
+:warning: This chart is currently under construction :warning:
+
+Please don't use it unless you're really sure about what you are doing.
```

Comment:
please migrate to the auto generated readme, see [admission-controller](https://github.com/sysdiglabs/charts/blob/master/charts/admission-controller/README.tpl) as example.